.github/workflows/cypress-repeat.yml

```diff
@@ -0,0 +1,66 @@
+name: Cypress repeat
+
+on:
+  workflow_dispatch:
+    inputs:
+      spec:
+        required: false
+        description: 'Spec to run. Leave empty if you prefer to filter tests using grep'
+      grep:
+        required: false
+        description: 'Run all tests which titles contain text. Leave empty if you prefer to filter tests using spec.'
+      dashboard_url:
+        required: true
+        default: 'https://automation-dashboard.staging.saleor.cloud/dashboard'
+        description: "Dashboard url"
+      API_url:
+        required: true
+        default: 'https://automation-dashboard.staging.saleor.cloud/graphql'
+        description: "API url"
+      repeat:
+        required: true
+        default: "10"
+        description: "How many times run tests"
+
+jobs:
+
+  run-tests:
+    if: ${{ github.event.inputs.spec }} || ${{ github.event.inputs.grep }} ## Do not run if spec or grep not provided
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
FYI they want to switch to v18 at https://github.com/saleor/saleor-dashboard/pull/2533 - you may need to change the CI if it's merged before your PR